src/users/users.controller.js

```diff
@@ -0,0 +1,26 @@
+import { UsersService } from "./users.service.js";
```

Comment:
Everything about the `users` live in the `users` module folder, this includes:

- `users.controllers.js` - the handlers that are part the http layer of the app
- `users.validator.js` - A file with very naive middleware that potentially checks the body of the requests
- `users.service.js` - The main business logic service that would take care of any more complicated business logic
- `users.repository.js` - The file that takes care of the data reading/writing, in this case the `.csv` file changes.

Reply:
The routes themselves are defined in the `/api` folder as requested.